package.json

```diff
@@ -17,9 +17,10 @@
     "inspectpack": "bin/inspectpack.js"
   },
   "scripts": {
-    "lint-server": "eslint --color -c .eslintrc-server lib bin",
+    "lint-server": "eslint --color -c .eslintrc lib bin",
```

Comment:
1. `-c .eslintrc` can be removed as it's default
2. Are we missing `lint-test` task and aggregation in `lint`?